less/components/popups.less

```diff
@@ -69,12 +67,54 @@
     align-items: center;
     width: 100%;
     height: 80px;
-    min-height: 80px;
+    flex-shrink: 0;
```

Comment:
I think the problems with flexbox were limited to how the top-level page template in `blueocean-web` was laid out, causing IE issues. Perhaps @tfennelly can clarify since I think he was the one who fixed them.